library/alloc/src/string.rs

```diff
@@ -1952,6 +1952,61 @@ impl String {
         Drain { start, end, iter: chars_iter, string: self_ptr }
     }
 
+    /// Converts a `String` into an iterator over the [`char`]s of the string.
+    ///
+    /// As a string consists of valid UTF-8, we can iterate through a string
+    /// by [`char`]. This method returns such an iterator.
+    ///
+    /// It's important to remember that [`char`] represents a Unicode Scalar
+    /// Value, and might not match your idea of what a 'character' is. Iteration
+    /// over grapheme clusters may be what you actually want. This functionality
```

Comment:
```suggestion
    /// over grapheme clusters may be what you actually want. That functionality
```

seems clearer to me this way, since "This" could be talking about this function or the grapheme clusters you referred to...

Reply:
OK. I'll update it. This is copied from the docs of `str::chars` so perhaps a follow-up to update those docs would be desired also.

Reply:
I think it's better to be consistent with `str::chars` than change something so minor.